src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/LiveMetricsTests.cs

```diff
@@ -467,12 +318,12 @@ public async Task TestSystemDiagnosticsMetrics_MaxTimeseries()
                         metricsConfiguration: new EventMetricsConfiguration
                         {
                             IncludeDefaultProviders = false,
-                            Providers = new[]
+                            Meters = new[]
                             {
-                                new EventMetricsProvider
+                                new EventMetricsMeter
                                 {
-                                    ProviderName = p1.ProviderName,
-                                    CounterNames = counterNames
+                                    MeterName = p1.ProviderName,
+                                    InstrumentNames = counterNames
```

Comment:
nit: `counterNames` -> `instrumentNames` and `providers` -> `meters` (applies to all tests)